lib/json.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+module.exports.register = function () {
+    let content
+    this
+        .on('documentsConverted', ({playbook, siteAsciiDocConfig, siteCatalog, uiCatalog, contentCatalog}) => {
+            console.log(playbook, siteAsciiDocConfig, siteCatalog, uiCatalog, contentCatalog)
+            content = contentCatalog.findBy({family: 'page'}).reduce((acc, item) => {
+                const urlVideo = item.asciidoc.attributes['cloud-url-video']
+                const category = item.asciidoc.attributes['cloud-category']
+                const order = item.asciidoc.attributes['cloud-order']
+                if (category) {
+                    acc.push({
+                        title: item.title,
+                        url: urlVideo ?? item.pub.url,
+                        category,
```

Comment:
should we have a list of allowed categories and validate against those?

Reply:
Yes, I can add this available list of categories to config